system-tests/lib/performance-reporter.js

```diff
@@ -1,157 +0,0 @@
-const path = require('path')
```

Comment:
was the reporter capturing test duration ?

Reply:
yes, and sending spans to honeycomb

https://ui.honeycomb.io/cypress/datasets/systemtest-performance/trace/GkrUXk1wjTY?fields[]=c_name&span=cfdb5416-e012-4406-ad67-2c4e3359180b

Reply:
cool 👍🏻 I assume we already captured this same info